www/runtest.php

```diff
@@ -1311,6 +1313,17 @@ function ValidateParameters(&$test, $locations, &$error, $destination_url = null
 
             if( !$test['aftEarlyCutoff'] && $settings['aftEarlyCutoff'] )
                 $test['aftEarlyCutoff'] = $settings['aftEarlyCutoff'];
+
+            if ( $test['lighthouse'] ) {
+              $lh_error = '';
+              if ( strpos($test['browser'], 'Chrome') === FALSE )
+                $lh_error = 'Lighthouse only works on Chrome';
+              if ( strlen($test['script']) )

```

Comment:
This was the only setting I immediately see will make the code paths completely divergent, everything else can likely be ignored, but can make them errors if desired.